cmd/yurt-app-manager/app/core.go

```diff
@@ -53,10 +54,10 @@ var (
 )
 
 func init() {
-	_ = clientgoscheme.AddToScheme(scheme)
 	_ = appsv1alpha1.AddToScheme(clientgoscheme.Scheme)
```

Comment:
this line can also be removed?

Reply:
Currently, it could not be removed, as yurtingress use it. We should refactor the code later.

Reply:
ok